working-with-fhir-igs/hl7v2-to-dk-medcom/mappings.bal

```diff
@@ -0,0 +1,180 @@
+import ballerina/uuid;
+import ballerinax/health.fhir.r4;
+import ballerinax/health.fhir.r4.international401;
+import ballerinax/health.hl7v2 as hl7;
+import ballerinax/health.hl7v23;
+import ballerinax/health.fhir.r4.medcom240;
+
+
+# Custom values can be populated using the incoing message. Result need to be merge with original resource.
+#
+# + originalMessage - incoming HL7v2 message
+# + return - Patient resource containing only the customly mapped values.
+public isolated function createCustomPatient(hl7:Message originalMessage) returns medcom240:MedComCorePatient|error {
```

Comment:
Let's use data mapping function for this, use local variables to store intermediate results needed

Reply:
Let's do all the mappings with transform patient function